integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLiftAndShiftFromOnPremDomain.java

```diff
@@ -76,7 +77,14 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.junit.jupiter.api.Assertions.fail;
 
-// Test to create model in image domain and verify the domain started successfully
+// Test to verify that wko managed weblogic domain can be created with the domain resource generated by
```

Comment:
Can you make this commented lines as class level  javadoc

Reply:
done